### DISCUSSION TITLE
use the vhEditor shortcuts with 'codeboard' apk from play store

### DISCUSSION THREAD

**Discussion by Ahmadreza-1995** [vhqtvn/VHSCode-Android#142] [Ideas]
1.install codeboard from this google play link :
[codeboard](https://play.google.com/store/apps/details?id=com.gazlaws.codeboard)
2.enable it from phone settings
3.go to vheditor user settings and change 'keyboard:dispatch' from 'code' to 'keycode'.
4. now you can use all vheditor shortcuts like vscode on desktop.
so you can customize your shortcuts.
add new or change defaults.... :)
* for use,dont press keys like hard keyboard , just tap on keys.its better than key pressing.
* codeboard public shortcuts will work in every apps that supports shortcuts. like copy,paste in browser's searchbar(eg : Ctrl+C for copy Selected text , Ctrl+Shift+v for Paste copied text).
https://user-images.githubusercontent.com/99149420/174473842-aec8e9a0-fd5f-4ac1-a77b-14a0a5aa454d.mp4

**Comment by vhqtvn** [comment]
The latest version with default vsboot.js should work with Hacker Keyboard.

**Comment by Ahmadreza-1995** [comment]
how can i download hacker keyboard? I cant find it in play store and other app stores or any where

**Comment by vhqtvn** [comment]
Current version has modified hacker keyboard builtin. Just go to settings and enable VHKeyboard in input methods.

**Reply by Ahmadreza-1995** [reply]
Thanks [<3]

**Comment by Ahmadreza-1995** [comment]
> Current version has modified hacker keyboard builtin. Just go to settings and enable VHKeyboard in input methods.

is hacker kbrd removed from latest version?

**Comment by BtAbdul** [comment]
unexpected keyboard is better than hacker keyboard, its specifically designed for programming, its open source keyboard.

**Comment by BtAbdul** [comment]
unexpected keyboard is better than hacker keyboard, its specifically designed for programming, its open source keyboard.

**Comment by BtAbdul** [comment]
unexpected keyboard is better than hacker keyboard, its specifically designed for programming, its open source keyboard.
